.vscode/launch.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
Please git ignore the `.vscode` directory. To do so:

1. Add `.vscode` to `.gitignore` file
2. `git rm --cached .vscode/launch.json`
3. `git add .`
4. `git commit`